components/profileCard/profileComponents.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components"
+
+export default function ProfileContainer() {
+  return (
+    <Container>
+      <Logo>
+        <img loading="lazy" src="/imagedotmenu.svg" alt="" />
+      </Logo>
+      <Wrapper>
+        <ImageContainer>
+          <img src="/image/ProfilePic.png" alt="" />
+        </ImageContainer>
+        <Span>Peter Parker</Span>
+        <PtAG>Joined in 2022</PtAG>
+      </Wrapper>
+    </Container>
+  )
+}
+
+const Container = styled.div`
+  width: 254px;
+  height: 331px;
+  background-color: #1a3185;
```

Comment:
You can use `var(--bs-blue)` to pull the color from our theme

Reply:
oh didn't know that